### DISCUSSION TITLE
问一下对于磁性材料的交换关联

### DISCUSSION THREAD

**Discussion by zc994646659** [deepmodeling/deepks-kit#66] [General]
在abacus 里面 是否可以设置不同的自旋构型来学习，以及 我的训练数据的胞要不要很大，因为实验上的磁结构是一个可能涉及到很大的胞的螺旋磁结构，目前的PBE描述的不是很好，另外问一下 这个机器学习需要的资源以及时长，需不需要用到 GPU加速。

**Comment by ouqi0711** [comment]
In principle, DeePKS model can be trained for systems with varied spin (although we have not tested such case before). Note that the band gap label is not supported for varied spin, but you may train the energy and force label. The training process is not that long and therefore the GPU acceleration is a necessity. 

P.S. English is desired on GitHub :)

**Comment by y1xiaoc** [comment]
Converting this to discussion as issues are more code related things
